src/api/routes.ts

```diff
@@ -2,6 +2,8 @@ import qs from 'qs';
 import { UUID } from '../types';
 
 export const ITEMS_ROUTE = 'items';
+export const ITEM_MEMBERSHIPS_ROUTE = 'item-memberships';
```

Comment:
Looks like you're mixing both `membership` and `memberships` for the same entity. We should probably make it consistent.

Reply:
There might be some confusions, but usually it's `membership`, except for `GET` as it fetches multiple of them.